openreview/profile/management.py

```diff
@@ -104,26 +104,38 @@ def set_remove_name_invitations(self):
             'name': {
                 'order': 1,
                 'description': 'Name that want to be removed.',
-                'value-regex': '.*',
-                'required': True                
+                'value': {
+                    'param': {
+                        'type': 'string',
+                        'regex': '.*'
+                    }
+                }
             },
             'usernames': {
                 'order': 2,
                 'description': 'Usernames that want to be removed.',
-                'values-regex': '~.*',
-                'required': True
+                'value': {
+                    'param': {
+                        'type': 'string[]',
+                        'regex': '~.*'
+                    }
+                }
             },
             'comment': {
                 'order': 3,
-                'value-regex': '[\\S\\s]{1,5000}',
                 'description': 'Reason why you want to delete your name.',
-                'required': False
+                'value': {
+                    'param': {
+                        'type': 'string',
+                        'maxLength': 5000,
+                        'markdown': True,
+                        'input': 'textarea',
+                        'optional': True
+                    }
+                }
             },
             'status': {
-                'order': 4,
-                'value-dropdown': ['Pending', 'Accepted', 'Rejected'],
-                'description': 'Request status.',
-                'required': True
+                'value': 'Pending'
```

Comment:
Why was this changed to a constant value?
Is it OK to remove description and order from the invitation?

Reply:
it is a constant value because there is another invitation that changes the status.